ddsp/training/decoders.py

```diff
@@ -160,7 +162,8 @@ def call(self, z_pitch, z_vel, z=None):
     if self.f0_residual:
       outputs['f0_midi'] += z_pitch
 
-    outputs['f0_hz'] = core.midi_to_hz(outputs['f0_midi'])
+    outputs['f0_hz'] = core.midi_to_hz(outputs['f0_midi'],
+                                       zero_silence=self.midi_zero_silence)
```

Comment:
maybe best to have the same kwarg name. I'd make `midi_to_hz` also have `midi_zero_silence` it's clearer

Reply:
Agree! Changed the zero_silence kwargs of midi_to_hz to midi_zero_silence.